src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  const getSum = function(firstNumber) {
+    count++;
+
+    return function(secondNumber) {
+      if (count > 3) {
+        return (count - 3) % 2 === 0
```

Comment:
this check can also be handled with `count > 3`
```suggestion
        if (count > 3 && count  % 2 === 0)
```